lib/pdn/Kconfig

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig PDN
+	bool "Packet Domain Network library"
+	depends on NRF_MODEM_LIB && AT_CMD && AT_NOTIF
+
+if PDN
+
+config PDN_CONTEXTS_MAX
+	int "Number of PDP contexts"
+	range 1 20
+	default 1
+	help
+	  Maximum number of PDP context handled by the library.
+	  This number does not include the default PDP context 0.
+
+config PDN_LEGACY_PCO
+	bool "Use legacy PCO mode"
+	help
+	  If selected, PCO will be used instead of ePCO during PDN connection establishment.
+
+config PDN_ESM_TIMEOUT
+	int "ESM timeout (ms)"
+	default 1000
+	help
+	  Timeout for waiting for an ESM notification when activating a PDN, in milliseconds.
+
+config PDN_SYS_INIT
+	bool "Initialize during SYS_INIT"
+
+if PDN_SYS_INIT
+
+config PDN_INIT_PRIORITY
+	int "Initialization priority"
+	default APPLICATION_INIT_PRIORITY
+
+config PDN_DEFAULTS_OVERRIDE
+	bool "Override defaults for PDP context 0"
+
+if PDN_DEFAULTS_OVERRIDE
+
+config PDN_DEFAULT_APN
```

Comment:
I find the naming a bit confusing. It seems that in option PDN_DEFAULTS_OVERRIDE the word "default" refers to the default parameters, but in the individual options PDN_DEFAULT_* the word "default" refers to the default bearer (PDP context 0), right?

Reply:
By default parameters I mean the configuration parameters of PDP context 0.
When `PDN_DEFAULTS_OVERRIDE` is enabled, the default configuration for PDP context 0 (APN, family etc) is overridden by `PDN_DEFAULT_APN`, `PDN_DEFAULT_FAM` and so on.